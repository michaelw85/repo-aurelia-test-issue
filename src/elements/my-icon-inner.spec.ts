import { bootstrap } from 'aurelia-bootstrapper';
import { ComponentTester, StageComponent, waitForDocumentElement } from 'aurelia-testing';

describe('Icon inner html setup', () => {
    let component: ComponentTester;

    describe('component', () => {
        beforeEach(() => {
            component = StageComponent.withResources('elements/my-icon-inner').inView('<my-icon-inner name="angle-up"></my-icon-inner>');
        });

        it('should render svg icon with href', async () => {
            await component.create(bootstrap);

            expect(await waitForDocumentElement('my-icon-inner')).toMatchSnapshot();
        });

        afterEach(() => component.dispose());
    });
});
