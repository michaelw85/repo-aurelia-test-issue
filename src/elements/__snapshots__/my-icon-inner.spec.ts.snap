// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon inner html setup component should render svg icon with href 1`] = `
<my-icon-inner
  au-target-id="2"
  class="au-target"
  name="angle-up"
>
  
  
  <svg
    au-target-id="1"
    class="au-target"
    innerhtml.bind="use"
  />
  

</my-icon-inner>
`;
